src/components/Catalog.vue

```diff
@@ -24,44 +24,66 @@
                                 <ul>
                                     <li v-for="(subdomain, index) in domain.subdomains" :key="index"> <div id="id_subdomain">{{subdomain.title}}</div>
                                         <ul>
-                                            <li id="list_skills" v-for="(skill, index) in subdomain.skills" :key="index">{{skill.title}}</li>
+                                            <li id="list_skills" v-for="(skill, index) in subdomain.skills" :key="index">
+                                                <input type="checkbox" id="checkbox" v-if="showCheckbox" v-model="registration.skills" v-bind:value="skill">
+                                                {{skill.title}}
+                                            </li>
                                         </ul>
                                     </li>
                                 </ul>
                             </v-card-text>
                         </v-card>
                     </v-flex>
                 </v-layout>
+                <v-btn class="btn_continue" v-if="showCheckbox" color="primary" @click="validate">Continue</v-btn>
             </v-container>
         </v-flex>
     </v-layout>
 </template>
 
 <script>
     import DomainService from '../services/domain.service';
+    import FormService from '../services/form.service';
+    import ApiService from '../services/api.service';
 
     export default {
         name: "Catalog",
+        props: {
+            showCheckbox: Boolean
+        },
         created() {
             // CatalogService.getAll().then((response) => {
             //     throw new Error(response.data);
             // }).catch((error) => {
             //     throw new Error(error.response.data);
             // });
-            this.domains = DomainService.getAll();
+            DomainService.getAll().then((response) => {

```

Comment:
Attention commenter ça et remettre : `this.domains = DomainService.getAll();` en attendant.